src/appleseed/renderer/kernel/lighting/pt/ptlightingengine.cpp

```diff
@@ -184,6 +187,7 @@ namespace
                     m_max_diffuse_bounces == ~0 ? "infinite" : pretty_uint(m_max_diffuse_bounces).c_str(),
                     m_max_glossy_bounces == ~0 ? "infinite" : pretty_uint(m_max_glossy_bounces).c_str(),
                     m_max_specular_bounces == ~0 ? "infinite" : pretty_uint(m_max_specular_bounces).c_str(),
+                    m_max_volume_bounces == ~0 ? "infinite" : pretty_uint(m_max_volume_bounces).c_str(),
```

Comment:
You forgot to update the format string, that's what Travis is complaining about.